include/spirv/unified1/spirv.core.grammar.json

```diff
@@ -4754,6 +4754,494 @@
       "extensions" : [ "SPV_KHR_shader_clock" ],
       "version" : "None"
     },
+    {
+      "opname" : "OpHitObjectRecordHitMotionNV",
+      "class"  : "Reserved",
+      "opcode" : 5249,
+      "operands" : [
+        { "kind" : "IdRef", "name" : "'Hit Object'" },
+        { "kind" : "IdRef", "name" : "'Acceleration Structure'" },
+        { "kind" : "IdRef", "name" : "'InstanceId'" },
+        { "kind" : "IdRef", "name" : "'PrimitiveId'" },
+        { "kind" : "IdRef", "name" : "'GeometryIndex'" },
+        { "kind" : "IdRef", "name" : "'Hit Kind'" },
+        { "kind" : "IdRef", "name" : "'SBT Record Offset'" },
+        { "kind" : "IdRef", "name" : "'SBT Record Stride'" },
+        { "kind" : "IdRef", "name" : "'Origin'" },
+        { "kind" : "IdRef", "name" : "'TMin'" },
+        { "kind" : "IdRef", "name" : "'Direction'" },
+        { "kind" : "IdRef", "name" : "'TMax'" },
+        { "kind" : "IdRef", "name" : "'Current Time'" },
+        { "kind" : "IdRef", "name" : "'HitObject Attributes'" }
+      ],
+      "capabilities" : [ "ShaderInvocationReorderNV" ],
+      "extensions" : [ "SPV_NV_shader_invocation_reorder" ],
```

Comment:
Entries that are not capabilities should only depend on capabilities, and not extensions.
Only a capability entry should depend on an extension.

Please remove the extensions entry everywhere except for the one at line 14430.

Sorry the CI doesn't catch this.
I have an outstanding task to fix that. https://github.com/KhronosGroup/SPIRV-Headers/issues/278